lib/clover_field.cpp

```diff
@@ -64,14 +125,11 @@ namespace quda {
     if (isNative()) bytes = 2*ALIGNMENT_ADJUST(bytes/2);
 
     // for twisted mass only:
```

Comment:
The comment `// for twisted mass only:` can be deleted here

Reply:
[4af48fe](https://github.com/lattice/quda/pull/1437/commits/4af48fe63bd1772bee115d7aa876fb45b610b19f)